playbooks/scenario46_sfsturbo.yaml

```diff
@@ -0,0 +1,136 @@
+        test_security_group_name: "{{ (prefix + '-test_sg_apimon') }}"
+        test_router_name: "{{ (prefix + '-test_router_apimon') }}"
+        test_network_name: "{{ (prefix + '-test_network_apimon') }}"
+        test_subnet_name: "{{ (prefix + '-test_subnet_apimon') }}"
+
+    - block:
+
+      - name: Get Availability zones
+        opentelekomcloud.cloud.availability_zone_info:
+        register: azs
+
+      - name: Set Availability Zone
+        set_fact:
+          az: "{{ azs.availability_zones.0.name }}"
+
+      - name: Create VPC (Reouter + Net + Subnet)
+        import_role:
+          name: opentelekomcloud.vpc
+        vars:
+          router_name: "{{ test_router_name }}"
+          network_name: "{{ test_network_name }}"
+          subnet_name: "{{ test_subnet_name }}"
+          state: present
+
+      - name: Get VPC Info
+        opentelekomcloud.cloud.vpc_info:
+          name_or_id: "{{ test_router_name }}"
+        register: vpc_info
+
+      - name: Get vpc id
+        set_fact:
+          vpc_id: "{{ vpc_info.vpcs.0.id }}"
+
+      - name: Get subnet info
+        openstack.cloud.subnets_info:
+          filters:
+            name: "{{ test_subnet_name }}"
+        register: subnet
+
+      - name: Get subnet ID
+        set_fact:
+          subnet_id: "{{ subnet.subnets.0.network_id }}"
+
+      - name: Create Security group
+        openstack.cloud.security_group:
+          name: "{{ test_security_group_name }}"
+          description: Test security group created by APImon
+
+      - name: Get SecurityGroup info
+        openstack.cloud.security_group_info:
+          name: "{{ test_security_group_name }}"
+        register: security
+
+      - name: Get SecurityGroup ID
+        set_fact:
+          security_group_id: "{{ security.security_groups.0.id }}"
+
+      - name: Create Security Group Rule
+        openstack.cloud.security_group_rule:
+          security_group: "{{ test_security_group_name }}"
+          protocol: tcp
+          port_range_min: 2048
+          port_range_max: 2052
+          remote_ip_prefix: 0.0.0.0/0
+
+      - name: Create sfs turbo share
+        opentelekomcloud.cloud.sfsturbo_share:
+          name: "{{ test_share_name }}"
+          share_proto: "NFS"
+          share_type: "STANDARD"
+          size: 100
+          availability_zone: "{{ az }}"
+          vpc_id: "{{ vpc_id }}"
+          subnet_id: "{{ subnet_id }}"
+          security_group_id: "{{ security_group_id }}"
+        register: share
+
+      - name: Expand capacity sfs turbo share
+        opentelekomcloud.cloud.sfsturbo_share:
+          name: "{{ test_share_name }}"
+          size: 200
+        register: share
+
+      always:
+
+        - block:
+
+           - name: Delete sfs turbo share
+             opentelekomcloud.cloud.sfsturbo_share:
+               name: "{{ test_share_name }}"
+               state: absent
+
+           - name: Pause for 60 seconds
+             pause:
+               seconds: 60
```

Comment:
please create issue on opentelekomcloud ansible-collection repo to introduce waiting for deletion

Reply:
https://github.com/opentelekomcloud/ansible-collection-cloud/issues/362